docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -78,49 +78,109 @@ Pick one of the following plugins:
 
 - [babel-plugin-import](https://github.com/ant-design/babel-plugin-import) with the following configuration:
 
+  `yarn add -D babel-plugin-import`
+
+  Create a `.babelrc.js` file in the root directory of your project:
+
   ```js
-  plugins: [
+  const plugins = [
     [
-      'babel-plugin-import',
+      "import",
       {
-        libraryName: '@material-ui/core',
-        libraryDirectory: 'esm', // or '' if your bundler does not support ES modules
-        camel2DashComponentName: false,
+        "libraryName": "@material-ui/core",
+        // Use '"libraryDirectory": "",' if your bundler does not support ES modules
+        "libraryDirectory": "esm",
+        "camel2DashComponentName": false
       },
-      'core',
+      "core"
     ],
     [
-      'babel-plugin-import',
+      "import",
```

Comment:
I think that it's better with the prefix. Better be explicit.

Reply:
I was just going with their documentation: https://github.com/ant-design/babel-plugin-import#usage

Reply:
I don't think that we should consider the convention of the plugin. I think that we should look at what Babel encourage. From what I remember, they started to encourage full names in v7.

Reply:
It seems that the shorthand is acceptable: https://babeljs.io/docs/en/plugins

Reply:
But maybe we should be explicit so people can more easily understand what's going on? This would be my only concern. What do you think?